Angular/src/app/project-form/project-form.component.html

```diff
@@ -5,13 +5,13 @@ <h2 mat-dialog-title>
   <mat-dialog-content>
     <mat-form-field>
       <input type="hidden" formControlName="id">
-      <input formControlName="name" matInput placeholder="Назва проекту*">
+      <input formControlName="name" required matInput placeholder="Назва проекту">
```

Comment:
add autofocus attribute
https://www.w3schools.com/tags/att_input_autofocus.asp
```suggestion
      <input formControlName="name" autofocus required matInput placeholder="Назва проекту">
```